components/app/header/ThemeSwitcher.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <label class="swap swap-rotate">
+    <input v-model="darkMode" type="checkbox" />
```

Comment:
issue (blocking): The input should use the full width of its parent, so it is triggered even if the parent is wider than the icon. The current implementation on mobile is problematic in regards to UX because the user may think the whole component is a button, when in reality it isn't.

Reply:
Adjusted ThemeSwitcher in a different way. The input already takes the entire width.